ale/base/__init__.py

```diff
@@ -1 +1,90 @@
 from ale.base.base import Driver
+
+spiceql_mission_map = {
+    "CHANDRAYAAN-1_M3": "m3",
+    "CHANDRAYAAN-1_MRFFR": "mrffr",
+    "CASSINI_ISS_NAC": "cassini",
+    "CASSINI_ISS_WAC": "cassini",
```

Comment:
It could go in spiceql, the translation has to exist somewhere. I think it's better off in ALE since ALE is using the spiceql interface rather than spiceql conforming to ALE.

Granted all of the keys are from NAIF so :shrug: